src/containers/Application/Part0.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { useLocation } from 'wouter'
+import VaccineInfo from '../../components/ApplicationForm/VaccineInfo'
+import NavigationButtons from '../../components/NavigationButtons'
+import VerticalProgressBar from '../../components/VerticalProgressBar'
+import { useAuth } from '../../utility/Auth'
+import { useHackerApplication } from '../../utility/HackerApplicationContext'
+import { checkForError, validateFormSection } from '../../utility/Validation'
+
+export default () => {
+  const { application, updateApplication, forceSave } = useHackerApplication()
+  const [, setLocation] = useLocation()
+  const { logout } = useAuth()
+  const [errors, setErrors] = useState({})
+  const [loading, setLoading] = useState(false)
+
+  const validate = change => {
+    const newErrors = validateFormSection(change, 'vaccineInfo')
+    setErrors({ ...errors, ...newErrors })
+    return { ...errors, ...newErrors }
+  }
```

Comment:
We're handling all validation in `validation.js`, not in the individual application parts. Refer to my comment above on adding a validator to the `vaccineInfo` field!
```suggestion
```